src/modules/75-cf/pages/environment-details/EnvironmentDetails.tsx

```diff
@@ -5,54 +5,126 @@
  * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
  */
 
-import React from 'react'
+import React, { useState } from 'react'
 import { useParams } from 'react-router-dom'
-import { Layout, Container, PageError } from '@wings-software/uicore'
+import { PageError } from '@harness/uicore'
+import routes from '@common/RouteDefinitions'
 import { ContainerSpinner } from '@common/components/ContainerSpinner/ContainerSpinner'
 import { useDocumentTitle } from '@common/hooks/useDocumentTitle'
 import type { EnvironmentResponseDTO } from 'services/cd-ng'
+import type { ApiKey } from 'services/cf'
+import { useGetAllAPIKeys } from 'services/cf'
```

Comment:
Duplicate import here

```suggestion
import { ApiKey, useGetAllAPIKeys } from 'services/cf'
```

Reply:
updated :)